docs/contributors/release-process.md

```diff
@@ -39,3 +48,13 @@ git rebase -i origin/master
 git tag -as v0.1 -m "Initial release."
 git push origin v0.1
 ```
+
+### After the release
+
+1. Announce the release on Twitter, Discord and other channels.
+2. Deploy the release:
+   - Inform clients
```

Comment:
Who are the "clients"? How one can inform them of the new release?

Reply:
Added a description of "clients" and some suggestions on how to contact them in [e029965](https://github.com/status-im/nwaku/pull/1001/commits/e0299657d286aae5b8a31340bf43f315115581f9). The idea is that this process will always be a bit fluid and often involves ad-hoc conversations with specific platform users and repo maintainers that might be specifically interested in this particular release.